pkg/apis/rollouts/v1alpha1/generated.proto

```diff
@@ -693,6 +693,15 @@ message GraphiteMetric {
   optional string query = 2;
 }
 
+// GraphiteMetric defines the Graphite query to perform canary analysis
```

Comment:
```suggestion
// InfluxdbMetric defines the InfluxDB Flux query to perform canary analysis
```

Reply:
This is actually expected, I fixed same graphite issues along the way.

Reply:
Actually my mistake, the graphite stuff I added is elsewhere - nice catch!